src/components/Description/style.less

```diff
@@ -0,0 +1,55 @@
+@import './../../styles/base.less';
+@sidemargin: 6vw;
+
+.description {
+  padding-top: 5em;
+  margin: 0 @sidemargin;
+  min-height: 100vh;
+  .description-content {
+    display: inline-flex;
+    align-items: center;
+    .description-text-container {
+      width: 60vw - @sidemargin;
+      padding-right: 5vw;
+      .description-text {
+        font-size: 1.5em;
+        padding-bottom: 1em;
```

Comment:
Could be shortened once you decrease the amount of wrapper divs

Reply:
(^･ω･^=)~